lib/components/sync-status/get-note-titles.ts

```diff
@@ -1,7 +1,7 @@
 import { compact } from 'lodash';
 import noteTitleAndPreview from '../../utils/note-utils';
 
-const getNoteTitles = (ids, notes, limit = Infinity) => {
+const getNoteTitles = (ids = [], notes = [], limit = Infinity) => {
```

Comment:
Can we avoid having multiple optional arguments following each other? This opens up a very odd situation where it looks like we can skip `ids` or `notes` when in reality it would just mush them together and break.

```ts
getNoteTitles(10); // ?
getNoteTitles([{id, data, version}, {id, data, version}]); // ?
```

it seems fair to enforce `ids` as `T.EntityId[]` and/or define `notes` to be `T.NoteEntity[]`. if we demand `notes: T.NoteEntity[]` though we should conditionally call this function. since `notes` in app state can be null it also makes sense to leave that open here…

```ts
const getNoteTitles = (ids: T.EntityId[], notes: T.NoteEntity[] | null, limit = Infinity) => {
  if (!notes) {
    return [];
  }

  …
}
```

Reply:
Probably something to handle over here: https://github.com/Automattic/simplenote-electron/pull/1894

Reply:
Handled in this commit: https://github.com/Automattic/simplenote-electron/pull/1894/commits/03b84ecc9a60b438a40a6075704724e77703e7a1